tests/CMakeLists.txt

```diff
@@ -53,12 +53,10 @@ list(APPEND LIT_ARGS "-Dxe_enabled=$<IF:$<BOOL:${XE_ENABLED}>,ON,OFF>")
 # ISPC enabled OS.
 list(APPEND LIT_ARGS "-Dwindows_enabled=$<IF:$<BOOL:${ISPC_WINDOWS_TARGET}>,ON,OFF>")
 list(APPEND LIT_ARGS "-Dmacos_arm_enabled=$<IF:$<BOOL:${ISPC_MACOS_ARM_TARGET}>,ON,OFF>")
-# Dumps enabled/disbaled
-list(APPEND LIT_ARGS "-Ddumps_enabled=$<IF:$<BOOL:${ISPC_NO_DUMPS}>,OFF,ON>")
 # ISPC opaque pointers
 list(APPEND LIT_ARGS "-Dopaque_mode=$<IF:$<BOOL:${ISPC_OPAQUE_PTR_MODE}>,ON,OFF>")
 add_custom_target(check-all DEPENDS ispc
-    COMMAND ${LIT_COMMAND} ${LIT_ARGS}
+    COMMAND ${LIT_COMMAND} ${LIT_ARGS} "--verbose"
```

Comment:
Please commit this line as a separate commit.

Arina and I recently discussed if we need to have `--verbose` by default and concluded that "probably yes". So please keep it in the PR. It's helpful for investigating problems in CI.

Reply:
Committed as separate. 

As I understand `--verbose` would dump detailed log only if some tests have failed.

Reply:
Yep.